P/PROJ/build_tarballs.jl

```diff
@@ -23,15 +24,18 @@ EXE_SQLITE3=${host_bindir}/sqlite3
 if [[ ${target} == *mingw* ]]; then
     SQLITE3_LIBRARY=${libdir}/libsqlite3-0.dll
     CURL_LIBRARY=${libdir}/libcurl-4.dll
-    TIFF_LIBRARY_RELEASE=${libdir}/libtiff-5.dll
+    TIFF_LIBRARY_RELEASE=${libdir}/libtiff-6.dll
 else
     SQLITE3_LIBRARY=${libdir}/libsqlite3.${dlext}
     CURL_LIBRARY=${libdir}/libcurl.${dlext}
     TIFF_LIBRARY_RELEASE=${libdir}/libtiff.${dlext}
 fi
 
-if [[ "${target}" == x86_64-linux-musl* ]]; then
-    export LDFLAGS="-lcurl"
+if [[ ${target} == x86_64-linux-musl ]]; then
+    # ${libdir}/libcurl.so needs a libnghttp, and it prefers to load /usr/lib/libnghttp2.so for this.
+    # Unfortunately, that library is missing a symbol. Setting LD_LIBRARY_PATH is not enough to avoid this.
```

Comment:
`LD_LIBRARY_PATH` is only for dynamic loading, if has nothing to do with linking, it isn't surpring it doesn't help anything here.